script.js

```diff
@@ -0,0 +1,426 @@
+    eyes: 'hidden',
+    accessories: ['glasses'],
+    other: []
+  },
+  {
+    name: 'Jaqueline',
+    img: 'images/jaqueline.svg',
+    hair: 'orange',
+    eyes: 'green',
+    accessories: ['glasses'],
+    other: ['necklace']
+  },
+
+  {
+    name: 'Jazebelle',
+    img: 'images/jazebelle.svg',
+    hair: 'purple',
+    eyes: 'hidden',
+    accessories: ['glasses'],
+    other: ['smoker']
+  },
+  {
+    name: 'Jean',
+    img: 'images/jean.svg',
+    hair: 'brown',
+    eyes: 'blue',
+    accessories: ['glasses', 'hat'],
+    other: ['smoker']
+  },
+  {
+    name: 'Jeane',
+    img: 'images/jeane.svg',
+    hair: 'brown',
+    eyes: 'green',
+    accessories: ['glasses'],
+    other: []
+  },
+  {
+    name: 'Jed',
+    img: 'images/jed.svg',
+    hair: 'orange',
+    eyes: 'green',
+    accessories: ['glasses', 'hat'],
+    other: ['smoker']
+  },
+  {
+    name: 'Jenni',
+    img: 'images/jenni.svg',
+    hair: 'white' ,
+    eyes: 'hidden' ,
+    accessories: ['hat'],
+    other: ['cellphone']
+  },
+  {
+    name: 'Jeri',
+    img: 'images/jeri.svg',
+    hair: 'orange',
+    eyes: 'green',
+    accessories: ['glasses'],
+    other: []
+  },
+  {
+    name: 'Jerry',
+    img: 'images/jerry.svg',
+    hair: 'hidden',
+    eyes: 'blue',
+    accessories: ['hat'],
+    other: []
+  },
+  {
+    name: 'Jess',
+    img: 'images/jess.svg',
+    hair: 'black',
+    eyes: 'blue',
+    accessories: ['glasses'],
+    other: []
+  },
+  {
+    name: 'Jocelyn',
+    img: 'images/jocelyn.svg',
+    hair: 'black',
+    eyes: 'brown',
+    accessories: ['glasses'],
+    other: []
+  },
+  {
+    name: 'Jon',
+    img: 'images/jon.svg',
+    hair: 'brown',
+    eyes: 'green',
+    accessories: ['glasses'],
+    other: []
+  },
+  {
+    name: 'Jordan',
+    img: 'images/jordan.svg',
+    hair: 'yellow',
+    eyes: 'hidden',
+    accessories: ['glasses', 'hat'],
+    other: []
+  },
+  {
+    name: 'Josephine',
+    img: 'images/josephine.svg',
+    hair: 'grey',
+    eyes: 'brown',
+    accessories: [],
+    other: []
+  },
+  {
+    name: 'Josh',
+    img: 'images/josh.svg',
+    hair: 'yellow',
+    eyes: 'green',
+    accessories: [],
+    other: []
+  },
+  {
+    name: 'Jude',
+    img: 'images/jude.svg',
+    hair: 'black',
+    eyes: 'green',
+    accessories: [],
+    other: []
+  },
+  {
+    name: 'Julie',
+    img: 'images/julie.svg',
+    hair: 'black',
+    eyes: 'brown',
+    accessories: ['glasses', 'hat'],
+    other: []
+  },
+]
+
+// All the DOM selectors stored as short variables
+const tilesSheet = document.querySelector('board');   
+const count = document.getElementById('counter');
+const filter = document.getElementById('filter')            //findout button
+const questions = document.getElementById('questions')
+const winOrLose = document.getElementById('winOrLose')
+const playAgain = document.getElementById('playAgain')
+const board = document.getElementById('board')
+const restartButton = document.getElementById('restart')
```

Comment:
Maybe move them to the top of the file to make it clearer to read the code and good choice of names when creating new DOM selectors.

Reply:
Hi Amanda, thank you for the constructive feedback.